script/rest/update-files.js

```diff
@@ -105,51 +107,11 @@ async function getBundledFiles(schemas) {
   // name of the `github/github` checkout. A CI test
   // checks the version and fails if it's not a semantic version.
   for (const filename of schemas) {
-    const schema = JSON.parse(await readFile(path.join(DEREFERENCED_DIR, filename)))
```

Comment:
const schema = JSON.parse(await readFile(path.join(DOCS_DEREF_OPENAPI_DIR, filename)))

     schema.info.version = `${githubBranch} !!DEVELOPMENT MODE - DO NOT MERGE!!`